src/main/java/gameboy/gamepack/racinggame/data/entity/Referee.java

```diff
@@ -0,0 +1,35 @@
+package gameboy.gamepack.racinggame.data.entity;
+
+import java.util.*;
+
+import gameboy.gamepack.racinggame.data.vo.RaceLog;
+
+public class Referee {
+
+    private static final int MIN_POSITION_VALUE = 0;
+
+    private RaceRecorder recorder;
+
+    public Referee() {
+        this.recorder = new RaceRecorder();
+    }
+
+    public void record(List<Car> cars) {
+        recorder.record(cars);
+    }
+
+    public List<RaceLog> playback() {
+        return recorder.playback();
+    }
+
+    public List<String> getWinnerNames() {
```

Comment:
지금은 우승자 이름을 반환하고 있는데 
요구사항이 변경되어 우승자의 이름과 나이를 출력해달라고 요청이 온다면 어떻게 될까요?

이러면 반환타입이 `List<String>`에서 `List<T>`로 변경되고
이 메소드를 사용하는 쪽도 변경되고 출력담당쪽도 변경되겠죠
이런 불상사를 막기 위해 객체를 넘겨주는 것이 좋습니다.

그러면 요구사항이 변경되어도 해당 객체에 필드만 추가하고
출력을 담당하는 곳에서는 해당 필드만 갈아끼우거나 추가해도되죠 

```suggestion
    public List<Car> getWinners() {
```